### ISSUE TITLE
[Accessibility] Browse By Tabs on Community/Collection Homepages are missing ARIA labels.

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#2700]
Originally reported in #1178 as part of the Deque Analysis in 2021.  This was rated as "serious" by Deque (one level down from "critical") which means:
> Serious: This issue results in serious barriers for people with disabilities, and will partially prevent them from accessing fundamental features or content. People relying on assistive technologies will experience significant frustration as a result. Issues falling under this category are major problems, and remediation should be a priority.

**Describe the bug**

This issue impacts **both** the Community and Collection homepages: 
* (ID 469552, 469655) "Selected state of the element is missing or incorrect." on the currently selected browse by option (e.g. "Subcommunities and Collections" on Community homepage, "Recent Submission" on Collection homepage). In both cases, the selected option is displayed visually, but not noted in ARIA tags.
    * Fix: Use the `aria-current` attribute to indicate the element that represents the current item within a container or set of related elements, such as the currently selected page, step, location, date, or time. https://www.w3.org/TR/wai-aria-1.1/#aria-current

**To Reproduce**
1. Can be reproduced on the demo site with these sample communities/collections:
    * https://demo.dspace.org/communities/0958c910-2037-42a9-81c7-dca80e3892b4
    * https://demo.dspace.org/collections/282164f5-d325-4740-8dd1-fa4d6d3e7200
2. When on either of those pages, verify that the Browse By tabs (under Collection/Community page title) do not have `aria-current` attributes.

**Related work**
Originally reported in #1178

**Comment by alexandrevryghem** [comment]
Atmire would like to claim this issue

**Comment by tdonohue** [comment]
@alexandrevryghem : Sounds great! Assigning to you then for you (or whoever you delegate to) to work on.

**Comment by tdonohue** [comment]
Ported to 7.6.2 in #2800
